packages/framework/src/Form/DatePickerType.php

```diff
@@ -31,6 +31,7 @@ public function configureOptions(OptionsResolver $resolver): void
             'widget' => 'single_text',
             'format' => static::FORMAT_PHP,
             'html5' => false,
+            'input' => 'datetime_immutable',
             'attr' => [
```

Comment:
Can we remove this option from form types where defined as it's now a default?
(i.e., PriceListFormType, ClosedDayFormType, ImportPriceListFormType)